lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -407,21 +412,25 @@ sub check_parameters {
 
   my @required_fields = qw/referenceName referenceBases assemblyId/;
 
-  if($parameters->{'alternateBases'}){
+  if($parameters->{'alternateBases'}) {
     push(@required_fields, 'alternateBases');
     push(@required_fields, 'start');
   }
-  else{
+  elsif($parameters->{'variantType'}) {
     push(@required_fields, 'variantType');
-    if($parameters->{'start'}){ 
+    push(@required_fields, 'start');
+    push(@required_fields, 'end');
```

Comment:
`start` and `end` are not required if `variantType` is used in a `geneId` query: http://docs.genomebeacons.org/variant-queries/#beacon-geneid-queries

Reply:
Same as previous comment.